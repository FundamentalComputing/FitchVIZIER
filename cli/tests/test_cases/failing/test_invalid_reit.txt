
1 | P
  | --
2 | Q   Reit: 1
