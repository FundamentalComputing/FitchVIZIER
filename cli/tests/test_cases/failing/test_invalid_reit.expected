Line 2: the proof rule Reit is used, but the sentence in this line is not the same as the sentence in the referenced line.